packages/wdio-allure-reporter/tests/allureFeatures.test.js

```diff
@@ -462,6 +461,21 @@ describe('hooks handling', () => {
         expect(endStep).toHaveBeenCalledTimes(1)
         expect(endStep.mock.results[0].value).toBe('failed')
     })
+
+    it('should ignore mocha all hooks if hook passes', () => {
+        reporter.allure = allureInstance()
+        reporter.onHookEnd({ title: '"before all" hook', parent: 'foo' })
+
+        expect(endStep).toHaveBeenCalledTimes(0)
+    })
+
+    it('should treat mocha all hooks as tests if hook throws', () => {
+        reporter.allure = allureInstance()
+        reporter.onHookEnd({ title: '"before each" hook', parent: 'foo', error: { message: '', stack: '' } })
+
+        expect(endStep).toHaveBeenCalledTimes(1)
```

Comment:
add expect(onTestStart) and expect(onTestFail) called

Reply:
How do I do that? Don't see such examples